### DISCUSSION TITLE
URLフラグメントがあるリンクで遷移するとヘッダーが見出しを隠す

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#881] [要望・アイデア・不具合]
### 前提

見出しの URL フラグメントが付されたリンクを踏んで Qiita を開けると、特定の見出しが先頭になるように表示されます。
また、Qiita はヘッダーが表示されます。

### 問題

見出しの URL フラグメントが付されたリンクで遷移すると、このヘッダーが見出しを隠してしまいます。

### 要望

ヘッダーが見出しを隠してしまうと、どこに遷移したのか瞬時に分かりません。見出しがヘッダーに隠れないようになっていると良いなぁと思います。

### 例

次の GIF では、記事「Markdown 記法 チートシート」の見出し「Format Text - テキストの装飾」に遷移するときの様子です。

![overlay-header](https://github.com/increments/qiita-discussions/assets/75878492/1c7ac138-a433-4cfe-8601-5a7ab22cf30f)

以下のリンクを踏んでいます。

https://qiita.com/Qiita/items/c686397e4a0f4f11683d#format-text---%E3%83%86%E3%82%AD%E3%82%B9%E3%83%88%E3%81%AE%E8%A3%85%E9%A3%BE

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！

一部の環境では再現していないため、お手数ですが下記を教えていただけると幸いです🙇
調査の上、対応したいと思います。

- OS・ブラウザの種類とバージョン

**Reply by yarakos95** [reply]
当方では以下の環境で再現されています。

- Windows 11 Home 
    - 23H2
- Microsoft Edge
    - 126.0.2592.102 (公式ビルド) (64 ビット)
- Chrome
    - 126.0.6478.127（Official Build）（64 ビット）

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
ありがとうございます！🙇

**Comment by makoto-ogata** [comment]
@Yarakashi-Kikohshi 
お待たせしました！
URLフラグメントがあるリンクで遷移しても、ヘッダー部分を表示させないように修正対応いたしました！
もしも、まだHeaderが表示され見出しを隠ししているようであれば、ブラウザの拡張機能などの可能性もありますので、拡張機能のオフもお試しいただけると幸いです 🙇🏻

**Reply by yarakos95** [reply]
確認が遅くなってしまいました
ヘッダが邪魔しないことを確認しました！対応ありがとうございます。

**Reply by tanaka-asami415** [reply]
こちらこそご確認いただきありがとうございました！
